leapp/reporting/__init__.py

```diff
@@ -0,0 +1,91 @@
+from leapp.models import Model, fields
+from leapp.topics import ReportTopic
+from leapp.libraries.stdlib.api import produce
+import json
+
+
+class Renderers(Model):
+    topic = ReportTopic
+
+    html = fields.String()
+    plaintext = fields.String()
+
+
+class Report(Model):
```

Comment:
For this and above, we would like now to use the docstrings too, to describe what the Model is used for and what it represents. I am fine with a follow up PR however please consider doing it.

Reply:
Above is "just" html and plaintext fields. Did I get it right or am missing something?

Reply:
I am talking about the model class docstrings - What the model is used for.

Reply:
done